### ISSUE TITLE
AVT 3 - Carbon X Link Component - Voices Over does not announce the disabled state of the disabled link

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2188]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description
Navigation to the disabled link. Voice Over Announces the link text but does not announce the state as disabled.
Note: To fix it add aria-disabled="true" to the disabled link.
Also, I recommend that you consider removing the last link because it is too long and does not follow the ARIA authoring practices or the Carbon usage documentation.
